api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -165,6 +167,49 @@ type VolumeReplicationGroupSpec struct {
 	// Action is either Failover or Relocate
 	//+optional
 	Action VRGAction `json:"action,omitempty"`
+	//+optional
+	KubeObjectProtection *KubeObjectProtectionSpec `json:"kubeObjectProtection,omitempty"`
+}
+
+type KubeObjectProtectionSpec struct {
+	// Preferred time between captures
+	// +optional
+	// +kubebuilder:validation:Format=duration
+	CaptureInterval *metav1.Duration `json:"captureInterval,omitempty"`
+
+	//+optional
+	CaptureOrder []KubeObjectsCaptureSpec `json:"captureOrder,omitempty"`
```

Comment:
Yes, right now the only difference is the "name" field's name, but in the near future we intend to support velero 1.9 https://github.com/RamenDR/ShioRamen/issues/58 which would cause them to diverge again with two fields unique to recovery time: `existingResourcePolicy` and `restoreStatus`